packages/swapper/src/types.ts

```diff
@@ -305,6 +306,10 @@ export type TradeQuoteStep = {
     chainflipChunkIntervalBlocks?: number
     chainflipMaxBoostFee?: number
   }
+  lifiSpecific?: {
+    lifiTools?: LifiTools
+    lifiRoute?: Route
```

Comment:
q: shoudn't these be defined and only `lifiSpecific` itself is optional?

Reply:
For those reasons:
- Aligned with other swappers
- Avoid casting the LifiTradeRate type (which doesn't exists anymore)

Safer at runtime but I agree we could probably do better at narrowing in the swapper